tests/wpt/metadata/custom-elements/custom-element-registry/define.html.ini

```diff
@@ -1,5 +1,53 @@
 [define.html]
   type: testharness
-  [Custom Elements: Element definition]
+  [If constructor is arrow function, should throw a TypeError]
```

Comment:
I suspect it's a limitation of our old version of Spider monkey. It would be worth seeing if Gecko passes that test.